src/document/mod.rs

```diff
@@ -84,113 +116,225 @@ impl DocumentEncryptOpts {
 }
 
 impl Default for DocumentEncryptOpts {
-    /// default to only sharing with the creator of the document
+    /// Constructs a `DocumentEncryptOpts` with common values.
+    ///
+    /// The document will have a generated ID and no name. Access to the document will be granted only to its creator.
```

Comment:
```suggestion
    /// The document will have a generated ID and no name. Access to decrypt the document will be granted only to its author.
```

Reply:
Reworded slightly, but took the suggestion.

I agree that "author" makes the most sense for describing who made the document, but we should keep in mind that `AssociationType` refers to it as `Owner`, which conveys something slightly different

Reply:
Now you made me sad that we aren't consistent with `author` and `owner` everywhere :(